docs/onenote/index.yml

```diff
@@ -64,7 +64,23 @@ landingContent:
             url: ../testing/test-debug-office-add-ins.md
           - text: Deploy and publish a OneNote add-in
             url: ../publish/publish.md
+            
+  # Card (optional)
+  - title: OneNote Developer Resources
+    linkLists:
       - linkListType: reference
         links:
+          - text: Ask questions
+            url: https://stackoverflow.com/questions/tagged/office-js
+          - text: Request features
+            url: https://feedbackportal.microsoft.com/feedback/forum/c06dcc30-2e1c-ec11-b6e7-0022481f8472
+          - text: Report issues
+            url: https://github.com/officedev/office-js/issues
+          - text: Join Office Add-ins community call
+            url: https://aka.ms/M365DevCalls
           - text: Office Add-ins additional resources
             url: ../resources/resources-links-help.md
+          - text: Join the developer community
+            url: https://pnp.github.io/
```

Comment:
Here and elsewhere, this pnp.github.io link goes to the Power Platform developer community, which isn't relevant for add-ins.

Reply:
Fixed in all three